src/scroll-timeline-css.js

```diff
@@ -0,0 +1,194 @@
+import { StyleParser, RegexMatcher } from "./scroll-timeline-css-parser";
+
+const parser = new StyleParser();
+
+function initMutationObserver() {
+  const sheetObserver = new MutationObserver((entries) => {
+    for (const entry of entries) {
+      for (const addedNode of entry.addedNodes) {
+        if (addedNode instanceof HTMLStyleElement) {
+          handleStyleTag(addedNode);
+        }
+        if (addedNode instanceof HTMLLinkElement) {
+          handleLinkedStylesheet(addedNode);
+        }
+      }
+    }
+
+    // TODO: Not a good idea to proxy all elements.
```

Comment:
This comment implies that you currently ARE proxying all elements. 

We can simply proxy element.style similar to how we proxy element.animate.  We take only a minor hit if setting a non-animation style on an element.  We accomplish this by swapping out Element.prototype.style.